server/pages/index.js

```diff
@@ -127,7 +127,8 @@ function getDefaultContext( request ) {
 		clientIp: request.ip ? request.ip.replace( '::ffff:', '' ) : request.ip,
 		isDebug: context.env === 'development' || context.isDebug,
 		tinymceWpSkin: context.urls[ 'tinymce/skins/wordpress/wp-content.css' ],
-		tinymceEditorCss: context.urls[ 'editor.css' ]
+		tinymceEditorCss: context.urls[ 'editor.css' ],
+		commitChecksum: getCurrentCommitShortChecksum()
```

Comment:
This will block the event loop for every request — because of `execSync` — which we don't want to do. Since it's not going to change for every request, we should stash this earlier, not in a function run in request handlers.